about.css

```diff
@@ -0,0 +1,519 @@
+@import url("https://fonts.googleapis.com/css?family=Lato");
+
+* {
+  font-family: "lato", sans-serif;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.hide {
+  display: none;
+}
+
+#header-list {
+  display: none;
+}
+
+.bar {
+  display: block;
+  width: 20px;
+  height: 4px;
+  margin: 5px auto;
+  margin-left: 5px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: black;
+}
+
+.nav-menu {
+  display: none;
+}
+
+.mobile-menu {
+  list-style-type: none;
+}
+
+.mobile-nav {
+  flex-direction: column;
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  margin-left: 0;
+  text-align: center;
+  transition: 0.3s;
+  padding-top: 10px;
+  background-color: rgba(236, 82, 66, 0.8);
+  -webkit-backdrop-filter: blur(8px);
+  backdrop-filter: blur(8px);
+  top: 0;
+  display: none;
+}
+
+.mobile-nav li {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 50px;
+  margin-left: -20px;
+  margin-top: 20px;
+  text-align: left;
+  color: #fff;
+}
+
+.diva {
+  color: #fff;
+  text-decoration: none;
+}
+
+.nav-toggle {
+  display: flex;
+  flex-direction: column;
+}
+
+.close-btn {
+  color: #fff;
+  font-size: 34px;
+  padding: 0 7px;
+  float: right;
+  margin-left: -85%;
+  margin-bottom: -40px;
+}
+
+.style-me {
+  display: none;
+  width: 50px;
+  margin-left: 20px;
+}
+
+.flez {
+  display: flex;
+}
+
+.headings {
+  font-family: "Lato", sans-serif;
+  color: black;
+}
+
+.small {
+  font-family: "Lato", sans-serif;
+  font-size: 35px;
+  text-align: center;
+}
+
+.main {
+  background-image: url("images/speaker-giving-talk.jpg");
+  background-color: grey;
+  background-repeat: none;
+  background-size: cover;
+}
+
+.big {
+  color: rgb(238, 66, 14);
+  font-size: 50px;
+  text-align: center;
+}
+
+.main-paragraph {
+  width: 100%;
+  margin-left: -97px;
+}
+
+.p1 {
+  color: black;
+  border: 3px solid #fff;
+  box-shadow: 0 5px 5px rgb(222 222 222 / 82%), 0 5px 5px rgb(222 222 222 / 82%);
+  padding: 20px 20px;
+  font-family: "Lato", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  margin-bottom: 40px;
+  width: 90%;
+  margin-left: 90px;
+}
+
+.linki {
+  color: black;
+  font-weight: 900;
+}
+
+.timing {
+  font-size: 30px;
+  font-weight: 900;
+  color: gray;
+}
+
+.contact {
+  text-align: center;
+}
+
+/* Summit section */
+
+.logo-boder {
+  text-align: center;
+  font-size: 15px;
+  color: black;
+  border: 3px solid #fff;
+  box-shadow: 0 5px 5px rgb(222 222 222 / 82%), 0 5px 5px rgb(222 222 222 / 82%);
+  padding: 20px 70px;
+  width: 20%;
+  margin-left: 150px;
+}
+
+.summit-head {
+  width: 100%;
+  text-align: center;
+}
+
+.summit-images1 {
+  background-color: rgb(238, 66, 14);
+  background-image: url("images/Summit1.jpeg");
+  background-repeat: none;
+  background-size: cover;
+  width: 90%;
+}
+
+.image-overlay {
```

Comment:
This style is not well-centered.
<img width="1554" alt="image" src="https://user-images.githubusercontent.com/31897434/227109340-847222f7-c52b-4d17-8a64-2b362ab31210.png">